tests/framework/installer/nfs_manifests.go

```diff
@@ -206,7 +208,69 @@ spec:
 
 // GetNFSServer returns NFSServer CRD instance definition
 func (i *NFSManifests) GetNFSServer(namespace string, count int, storageClassName string) string {
-	return `apiVersion: v1
+	return `kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nfs-client-provisioner-runner
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
```

Comment:
i think this change only does GET and CREATE with storageClasses.  Although since there isn't an integration test that exercises the dynamic provisioning flow, we should keep these RBAC definitions in agreement with the example YAML.

Reply:
the LIST is required as well for the provisioner. I tried removing list but it didn't worked. Yeah it is in sync with the examples.